src/chrono_cache.rs

```diff
@@ -0,0 +1,89 @@
+use std::hash::Hash;
+use std::sync::{Arc, RwLock};
+use std::time::Duration;
+
+use ttl_cache::TtlCache;
+
+#[derive(Clone)]
+pub struct ChronoCache<K: Hash + Eq, V> {
+  data: Arc<RwLock<TtlCache<K, V>>>,
+}
+
+impl<K: Hash + Eq, V> std::fmt::Debug for ChronoCache<K, V> {
+  fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+    write!(f, "ResCache {{capacity: {:?}}}", self.data.read().unwrap().capacity())
+  }
+}
+
+impl<K: Hash + Eq, V: Clone> Default for ChronoCache<K, V> {
+  fn default() -> Self {
+    Self::new()
+  }
+}
+
+impl<K: Hash + Eq, V: Clone> ChronoCache<K, V> {
+  pub fn new() -> Self {
+    ChronoCache { data: Arc::new(RwLock::new(TtlCache::new(1000))) }
+  }
+
+  #[allow(clippy::too_many_arguments)]
+  pub fn insert(&self, key: K, value: V, ttl: u64) -> Option<V> {
+    let ttl = Duration::from_secs(ttl);
+    self.data.write().unwrap().insert(key, value, ttl)
+  }
+
+  pub fn get(&self, key: &K) -> Option<V> {
+    self.data.read().unwrap().get(key).cloned()
+  }
+
+  #[allow(dead_code)]
+  pub fn remove(&self, key: &K) -> Option<V> {
+    self.data.write().unwrap().remove(key)
+  }
+}
+
+#[cfg(test)]
+mod tests {
+
+  // #[test]
```

Comment:
do we need that tests? Looks like we have a thin wrapper to library that is probably has its tests

Reply:
removed the tests